### ISSUE TITLE
Gater option x2 /2

### ISSUE THREAD

**Issue by dmt-hub** [esaruoho/org.lackluster.Paketti.xrnx#368]
would it be possible to have an option in gater which you can click x2 and it puts the whole song automatically (so everything sounds the same) in LPB 8 instead of LPB 4, and the gater sounds double as fast.

it autmatically stretches your patterns so the beats still sound at same bpm, only gater becomes double as fast.

so
x2 -> lpb4 -> lpb8  / pattern length changes (ie. 32->64) -> pattern stretched from 32 -> 64
/2 -> lpb8 -> lpb4  / pattern length change (64->32) -> pattern shrinked 64 -> 32

so basically these 2 shortcuts in button form inside gater
![48975](https://github.com/user-attachments/assets/f77e9755-af47-44b0-978c-a09c49288b66)

**Comment by dmt-hub** [comment]
implemented! thanks!
